extraDevfileEntries.yaml

```diff
@@ -0,0 +1,18 @@
+schemaVersion: 1.0.0
+samples:
+  - name: nodejs-basic
+    displayName: Basic NodeJS
+    description: A simple Hello World application
+    icon: nodejsIcon.svg
+    tags: ["NodeJS", “Express”]
```

Comment:
Let's add `language` and `projectType` to the list (to match the stack list):
For `nodejs-basic`,
```
  projectType: nodejs,
  language: nodejs,
```

For vertx-secured-http,
```
  projectType: vertx,
  language: java, 
```

Reply:
Good suggestion, will update.

Reply:
Done